content/guides/virtualizing/hyper-v.md

```diff
@@ -5,68 +5,176 @@ date = "2013-01-14T10:32:53.000Z"
 tags = []
 +++
 
-There are several methods offered for installing Haiku. These include RAW images, ISO images and Anyboot images; which are a hybrid of the latter two images. The RAW images are pre-installed environment, in which the Virtual Hard Disk size cannot be customized, but going through the installation process is not required. The ISO images on the other hand can be burnt to a writeable CD or DVD, but can used in hyper-V to create a virtual CD-ROM that can be used to install Haiku onto a virtual Hard Disk. This guide will show you how to get Haiku up and running in Microsoft’s Hyper-V using the ISO images. 
+There are two methods for installing Haiku: RAW images and Anyboot images. RAW images consist of a pre-installed environment, whereby the Virtual Hard Disk is of a fixed size. By using a RAW image, you do not need to go through the installation process.
```

Comment:
The anyboot image can act as both a raw hard drive image or a ISO CD image, depending on your have it attached to a virtual hard drive or a virtual CD-ROM drive.

Reply:
I have tried to update the files based on your suggestion.